base/steer/FairRootManager.cxx

```diff
@@ -71,7 +71,8 @@ FairRootManager* FairRootManager::Instance()
 {
```

Comment:
With this PR no user code is broken at all, the IO Manager is created by the Framework and the user are accessing it via the FairRootManager::Instance() which is still returning a pointer as before. I think if we decide to change all the other Singletons (which will definitely break the user code!) then we improve this further.